src/pages/Browse/index.tsx

```diff
@@ -119,6 +130,16 @@ export class BrowseBase extends React.Component<Props> {
         }),
       );
     }
+
+    if (file && nextFilePath && !nextFileIsLoading && nextFile === undefined) {
```

Comment:
Do you think this really needs to check `if (file...)`? I removed that and played with the UI. Everything still seems to work. 

If you were to remove that check, I think the preload would happen in parallel with the initial load. This would be a good thing. It would give the UI a better chance at keeping up with someone who is pressing `j` quickly. 

But, I dunno, maybe it only applies to the first time you load a file. In any case, if you can delete that part of the if statement you could delete one of the tests and that's good.

Reply:
Will fetching next file in parallel make it slower to get the first file (when Internet speed is slow)? My original thought is not to make the initial loading slower due to the preload.

Reply:
Even though JS is event driven I guess it could tie up the network interface, yeah. Good point. Let's keep it as is.